packages/cli/src/lib/config.ts

```diff
@@ -29,22 +29,34 @@ const LEGACY_CONFIG_FOLDER = join(homedir(), '.llama-pack');
 const LEGACY_CONFIG_NAME = 'llama-pack-config.json';
 const LEGACY_CONFIG_FILE_PATH = join(LEGACY_CONFIG_FOLDER, LEGACY_CONFIG_NAME);
 
-async function createConfig(): Promise<Config> {
-  const result = await inquirer.prompt([
-    {
-      name: 'jdkPath',
-      message: 'Path to the JDK:',
-      validate: existsSync,
-    }, {
-      name: 'androidSdkPath',
-      message: 'Path to the Android SDK:',
-      validate: existsSync,
-    },
-  ]);
-  return new Config(result.jdkPath, result.androidSdkPath);
+async function validateSdkInput(path: string): Promise<Result<string, Error>> {
+  if (!existsSync(path) || !existsSync(join(path, 'build-tools'))) {
+    return Result.error(Error('Invalid path. Run \'bubblewrap doctor for more information\''));
+  }
+  return Result.ok(path);
+}
+
+async function validateJdkInput(path: string): Promise<Result<string, Error>> {
+  if (!existsSync(path)) {
+    return Result.error(Error('Invalid path. Run \'bubblewrap doctor for more information\''));
+  } else {
+    const file = await fsPromises.readFile(join(path, 'release'), 'utf-8');
+    if (!file || (file.indexOf('JAVA_VERSION="1.8') < 0)) {
+      return Result.error(Error('Invalid path. Run \'bubblewrap doctor for more information\''));
+    }
+  }
+  return Result.ok(path);
 }
 
-async function renameConfigIfNeeded(log = new Log('config')): Promise<void> {
+
+async function createConfig(prompt: Prompt = new InquirerPrompt): Promise<Config> {
+  // const jdkPath = prompt.promptInput()
+  const jdkPath = prompt.promptInput('Path to the JDK:', null, validateJdkInput);
```

Comment:
We should add `await` when calling the function instead of when using the value.